### ISSUE TITLE
"In times of change learners inherit the earth" ~ Eric Hoffer

### ISSUE THREAD

**Issue by nelsonic** [dwyl/quotes#23]
This is a _superb_ quote about continuous learning:

# "In times of change learners inherit the earth, while the learned find themselves beautifully equipped to deal with a world that no longer exists." ~ Eric Hoffer

# Todo:

+ [x] Create Pull Request to add the following quote to the `quotes.json` list:

```json
  {
    "author": "Eric Hoffer",
    "source": "https://www.goodreads.com/quotes/10562",
    "tags": "change, learners, learned, learn, world",
    "text": "In times of change learners inherit the earth, while the learned find themselves beautifully equipped to deal with a world that no longer exists."
  },
```
Discovered via @cristianofromagio ❤️ via https://github.com/dwyl/javascript-todo-list-tutorial/pull/27 :shipit: 
<img width="704" alt="cristianofromagio-quote" src="https://user-images.githubusercontent.com/194400/80411276-d7475d00-88c3-11ea-87ca-85c098ead20f.png">

**Comment by iteles** [comment]
Closed with PR #24. Thanks @nelsonic and @sahilister

**Comment by cristianofromagio** [comment]
Hey, it's me! 😄 Glad I could help somehow with the quote but not much with the pull request. Anyway, thanks @sahilister for the help and @nelsonic & @iteles for the awesome dwyl community. You rock!
